### ISSUE TITLE
Color contrast of tree view selected item and its border is less than 3:1

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#1535]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.67.2
- OS Version: Windows 11

Steps to Reproduce:

1. Go to VS Code explorer 
2. Select an item on tree. 
3. Observe color contrast with selected item and its border

Expected : Color contrast of keyboard focus on the selected item should be than 3:1.

Actual :  Color contrast is less than 3:1.

<img width="1004" alt="image" src="https://user-images.githubusercontent.com/37163639/172743042-da6d4ba0-2f4c-4770-8f14-de72fd9ac9a6.png">
